code/src/components/CompleteMessage.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+const CompleteMessage = () => {
```

Comment:
Nice that you added a message when tasks are completed! Another way could be to move this in to the AllTodoItems-component. And there you could do some conditional rendering so that if the to-do-list is empty the message appears. This is of course just a suggestion.